crates/reth/evm/src/precompiles/schnorr.rs

```diff
@@ -0,0 +1,65 @@
+use std::array::TryFromSliceError;
+
+use revm::primitives::{PrecompileError, PrecompileOutput, PrecompileResult};
+use revm_primitives::Bytes;
+use strata_crypto::verify_schnorr_sig;
+use strata_primitives::buf::{Buf32, Buf64};
+use thiserror::Error;
+
+/// 32 Bytes: Public key
+/// 32 Bytes: Message hash
+/// 64 Bytes: Schnorr Signature
+struct SchnorrInput {
+    public_key: Buf32,
+    message_hash: Buf32,
+    signature: Buf64,
+}
```

Comment:
This is wrong.
You should add that info for the fields and the docstring for the `struct` should explain the `struct` itself.